docs/ref/models/fields.txt

```diff
@@ -73,6 +73,12 @@ purely database-related, whereas :attr:`~Field.blank` is validation-related. If
 a field has ``blank=True``, form validation will allow entry of an empty value.
 If a field has ``blank=False``, the field will be required.
 
+.. tip:: Supplying missing values
+
+    ``blank`` can still be used with fields having ``null=False``, but this
+    necessitates implementing :meth:`~django.db.models.Model.clean` to
```

Comment:
I'd personally go with _requires_ rather than _ necessitates_ for simplicity of language.

I kind of also think it should be `implementing Model.clean on your model class` (or something less stilted), otherwise I'm questioning why I have to implement Model.clean? Shouldn't Model already have that? Do I have to edit the Django source? [If I click through the link, I'm sure all will be explained...]